.github/workflows/build-vcpkg.yml

```diff
@@ -117,6 +117,35 @@ jobs:
       asset-name: 'docker-ubuntu-22_04'
     secrets: inherit
 
+  check-documentation-changes:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    runs-on: ubuntu-22.04
+    outputs:
+      documentation_contents_changed: ${{ steps.variables.outputs.documentation_contents_changed }}
+    steps:
+      - name: Check for Documentation Changes
+        id: changed
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
+            src:
+              - 'docs/**'
+      - name: Set Output
+        id: variables
+        run: |
+          echo "documentation_contents_changed=${{ steps.changed.outputs.src }}" >> $GITHUB_OUTPUT
+      - name: Print Variables
+        run: |
+          echo "${{ toJSON(steps.variables.outputs)}}"
```

Comment:
Can you open a new Jira ticket to apply the `dorny/paths-filter@v3` to some of our other build paths.

Note:  It might be worth adding any changes to the ".github" folder as a condition to test/build all...

Reply:
https://hpccsystems.atlassian.net/browse/HPCC-32370